README.md

```diff
@@ -47,6 +47,7 @@ This generator can also be further configured with the following command line fl
         --pug           add pug engine support
     -H, --hogan         add hogan.js engine support
     -v, --view <engine> add view <engine> support (dust|ejs|hbs|hjs|jade|pug|twig|vash) (defaults to jade)
+        --no-view       use HTML directly instead of a view engine
```

Comment:
I wonder if it wouldn't be useful to mention API services as well. Something like this:
```--no-view       use HTML directly instead of a view engine, or either build an API service```

Reply:
I'll leave that up to @dougwilson's discretion, I'm happy with whatever help text makes sense.